tests/test_report.py

```diff
@@ -68,14 +71,17 @@ def test_make_report_open(tmp_dir, mocker):
     assert not mocked_open.called
 
     mocked_open = mocker.patch("webbrowser.open")
+
     live = Live(report=None)
     live.log("foo", 1)
     live.next_step()
 
     assert not mocked_open.called
 
+    monkeypatch.setenv(DVCLIVE_OPEN, True)
+
     mocked_open = mocker.patch("webbrowser.open")
```

Comment:
Do we need to patch it a second time? Seems like the one from line 73 should be enough.

Reply:
We don't, updated!